features/admin.authentication.v1/utils/authenticate-utils.ts

```diff
@@ -73,7 +73,7 @@ export class AuthenticateUtils {
             responseMode: window["AppUtils"]?.getConfig()?.idpConfigs?.responseMode ?? responseModeFallback,
             scope: window["AppUtils"]?.getConfig()?.idpConfigs?.scope ?? [ TokenConstants.SYSTEM_SCOPE ],
             sendCookiesInRequests: true,
-            sessionRefreshInterval: window[ "AppUtils" ]?.getConfig()?.session?.sessionRefreshTimeOut,
+            sessionRefreshInterval: -1,
```

Comment:
Is this expected?

Reply:
This is to disable the session refresh from SDK. We can't pass undefined as SDK has a fallback value defined.

Reply:
There should be an option in the SDK to disable session refresh interval right?
Shall we create an issue in the SDK for this and move forward with the workaround for the timebeing.

Reply:
Created an issue: https://github.com/asgardeo/asgardeo-auth-react-sdk/issues/226